### ISSUE TITLE
Fatal error:  Cannot instantiate interface Magento\Framework\Filesystem\DriverInterface

### ISSUE THREAD

**Issue by benjiam-gh** [webcodebg/glami#1]
Hello,
trying to install version 103.0.1 on Magento 2.3.2 the following error occurs:
```
PHP Fatal error:  Uncaught Error: Cannot instantiate interface Magento\Framework\Filesystem\DriverInterface in /opt/bitnami/apps/magento/htdocs/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php:50
Stack trace:
#0 /opt/bitnami/apps/magento/htdocs/vendor/magento/framework/ObjectManager/ObjectManager.php(70): Magento\Framework\ObjectManager\Factory\Dynamic\Developer->create('Magento\\Framewo...')
#1 /opt/bitnami/apps/magento/htdocs/vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php(160): Magento\Framework\ObjectManager\ObjectManager->get('Magento\\Framewo...')
#2 /opt/bitnami/apps/magento/htdocs/vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php(246): Magento\Framework\ObjectManager\Factory\AbstractFactory->resolveArgument(Array, 'Magento\\Framewo...', NULL, 'filesystemDrive...', 'Webcode\\Glami\\S...')
#3 /opt/bitnami/apps/magento/htdocs/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php(34): Magento\Framework\ObjectManager\Factory\AbstractFactory->resolveArgum in /opt/bitnami/apps/magento/htdocs/vendor/magento/framework/ObjectManager/Factory/Dynamic/Developer.php on line 50
```
Thanks

**Comment by Bashev** [comment]
Do you try to delete generated folder and run setup:di:compile

**Comment by benjiam-gh** [comment]
@Bashev nothing.
After clear generated directories and added glami module:
composer require webcodebg/module-glami:^103.0
the problem occurs during:
magento setup:upgrade

The plugin problem looks like this:
https://github.com/MultiSafepay/magento2/issues/2

**Comment by Bashev** [comment]
Fixed in version 104.1.0
